Automattic/wp-calypso
# Add newsletter badge for newsletter plan

Closes https://github.com/Automattic/wp-calypso/issues/74293

## Proposed Changes
Add a newsletter badge to newsletter payment plans so that users can easily identify a newsletter payment plan from other plans.


<img width="1169" alt="Screenshot 2023-03-09 at 1 41 50 PM" src="https://user-images.githubusercontent.com/7076981/224130294-8af44ec2-acd2-475b-82e5-1ef26b6ddf03.png">

## Testing Instructions



1. Go to `http://calypso.localhost:3000/earn/payments-plans/${siteSlug}`
2. Add a payment plan and be sure the "Email newly published posts to your customers."  option from the "Email" tab is selected.
3. Save the plan and verify the "Newsletter" badge is shown
4. Repeat steps 2 without selecting the "Email newly published posts to your customers."  option from the "Email" 
5. The "Newsletter" badge should not be shown

## Pre-merge Checklist
